frontend/src/pages/Player/PlayerHook/PlayerHook.tsx

```diff
@@ -799,14 +796,6 @@ export const usePlayer = (): ReplayerContextInterface => {
 		state.sessionMetadata,
 	])
 
-	useEffect(() => {
-		const searchParamsObject = new URLSearchParams(location.search)
-		if (searchParamsObject.get(PlayerSearchParameters.errorId)) {
-			setShowLeftPanel(false)
-			setShowRightPanel(true)
-		}
-	}, [setShowLeftPanel, setShowRightPanel])
```

Comment:
Not showing the error in the right panel anymore because of related resource panel. Also, we're using the `PlayerSearchParameters.search` param to determine whether we should close the left panel now, so decided to clean up this logic entirely.